doc/io_streams.rdoc

```diff
@@ -288,7 +358,7 @@ in the stream where the next read will occur.
 
 A new stream is initially has line number +0+.
 
-\Method IO#lineno returns the line number.
+\Method IO#lineno (also in \ARGF, \File, and \StringIO) returns the line number.
```

Comment:
I don't think we need to mention that this is available in ARGF, File, StringIO since we initially said:

> Except as noted, the instance methods described on this page are available in classes ARGF, File, IO, and StringIO.

Reply:
Removed.